### PR TITLE
chore(docs): Change pronouns in HTML generation

### PR DESCRIPTION
**Pull request by KillMonger1**
## Description

Changed the pronouns from "we" to "you" and "our" to "your" in html-generation.md
## Related Issues

Addresses #18284

### CODE REVIEW

**Comment by KillMonger1** [comment]
@marcysutton @gillkyle Please let me know if any changes are required.

File: docs/docs/html-generation.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The high level process is:
 
 ## Webpack
 
-For the first step, we use webpack to build an optimized Node.js bundle. The entry point for this is called `static-entry.js`
+For the first step, Use webpack to build an optimized Node.js bundle. The entry point for this is called `static-entry.js`
```
Comment (TylerBarnes) [inline docs/docs/html-generation.md]:
```suggestion
For the first step, use webpack to build an optimized Node.js bundle. The entry point for this is called `static-entry.js`
```

**Comment by KillMonger1** [comment]
@marcysutton Please review my PR.

**Comment by KillMonger1** [comment]
Hey @sidharthachatterjee. Could you Please review my PR ?

**Comment by LekoArts** [comment]
@KillMonger1 Please do **not** ping people (repeatedly) especially on weekends. We work from Monday till Friday and will handle your PR as everyone else’s. I'd consider pinging people every day to be rude. Thanks for understanding!

**Comment by LekoArts** [comment]
Unfortunately we can't merge this PR as-is because the "we" here refers to Gatsby, not the reader as also mentioned here: https://github.com/gatsbyjs/gatsby/pull/18428

> The subject of this doc is more about Gatsby than the reader; we don't want readers to be confused about what they have to do manually. I'll make a note to update the Style Guide to make this more explicit.

So it's not simply a process of converting "we" to "you", please have a look at the changes that Marcy requested to get an idea of how you'd need to change the document, e.g. the comment:

> It's somewhat subjective how to structure sentences, but I agree that this doesn't quite work here (the language used was odd to begin with). I'd recommend restructuring this and the following paragraphs to not talk about "talking", but rather present the information more directly. In some cases "we" means Gatsby's code, so it would be more appropriate to talk about the code as a subject since it's doing the action (not the user/developer).

Please let us know if you need help with that.

**Comment by KillMonger1** [comment]
I'm really sorry. This was my first PR. Sorry for the inconvenience caused

**Comment by KillMonger1** [comment]
So shall I try restructuring it?

**Comment by LekoArts** [comment]
Sure, if you feel comfortable doing it go ahead! 👍

File: docs/docs/html-generation.md [hunk 2]
```diff
@@ -91,14 +91,14 @@ The [production-app.js](/docs/production-app/#first-load) needs to know the page
 
 #### 7. Render Final HTML Document
 
-Finally, we call [react-dom](https://reactjs.org/docs/react-dom.html) and render our top level Html component to a string and return it.
+Finally, you call [react-dom](https://reactjs.org/docs/react-dom.html) and render our top level Html component to a string and return it.
```
Comment (marcysutton) [inline docs/docs/html-generation.md]:
```suggestion
Finally, you call [react-dom](https://reactjs.org/docs/react-dom.html) and render our top level HTML component to a string and return it.
```
Might as well fix this while you're in there!

**Comment by LekoArts** [comment]
@marcysutton I disagree that it would be good to merge as written in my post above.

For example changing this sentence to:

> For the first step, use webpack to build an optimized Node.js bundle. The entry point for this is called `static-entry.js`

The end-user doesn't use webpack, Gatsby does. And so on in the other changes.

**Comment by marcysutton** [comment]
Oh I see, you’re right @LekoArts. The Style Guide does talk about that now: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun

**Comment by marcysutton** [comment]
Hi @KillMonger1, are you able to address the changes so we can merge this in time for Hacktoberfest?

**Comment by LekoArts** [comment]
Let's close this PR then as it seems stale and the requested changes are larger than a couple of quick fixes.

**Pull request closed without merging**